### ISSUE TITLE
Internal typings dependency tree is disconnected

### ISSUE THREAD

**Issue by lildeadprince** [ReactiveX/rxjs#6175]
## Bug Report
May be related to #3674 #4511 #4512 #4540 
I'm opening the issue since they're either long time closed or have crazy ridiculous advices like "just downgrage, man". And also as a place to describe a possible temporary fix for those who might've had a similar setup.

**Current Behavior**
Import sub-component like 'rxjs/webSocket', but do not import 'rxjs'.
Partial usage of sub-components does not import all the dependent typing causing into compilation errors. 

**Expected behavior**
If some typing requires some global interface being extended, then this extension should be explicitly defined.
I.e.: `type InteropObservable<T> = { [Symbol.observable]: () => Subscribable<T>; }` should explicitly import a module which declares `Symbol.observable` , and so on.
(I am not aware of any other cases, but I do not know whether they exist or not)

**Reproduction**

```typescript
import { webSocket as rxSocket } from 'rxjs/webSocket'
export default (url, WebSocketCtor) => rxSocket({ url, WebSocketCtor })
```
Essentially the import itself will cause the issue but I've added the second line (the usage) just for a convenience. Personally I used it with `reconnecting-websocket` package (in which funnily I also had to patch some incorrect types).

The error is following
```
node_modules/rxjs/internal/types.d.ts:45:13 - error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.

45     [Symbol.observable]: () => Subscribable<T>;
               ~~~~~~~~~~
```

**Environment**
- Runtime: Node 15.11
- RxJS version: 6.6.6

**Possible Solution**

The actual cause is obvious: the declaration of `Symbol.observable` was not imported.
Brief search pinpoints the Rx.d.ts file that contains the required declaration.

The minimal solution is as simple as that:
```typescript
import 'rxjs'; // just add this line and all necessary typings will be imported
import { webSocket as rxSocket } from 'rxjs/webSocket'
export default (url, WebSocketCtor) => rxSocket({ url, WebSocketCtor })
```

The workaround is simple, but it is still a workaround and the presence of the issue detects some internal inconsistency in the typings.
So for an actual fix as I mentioned above it'd be better to extract the "global" declarations somewhere and import them where necessary.

**Comment by cartant** [comment]
Could you include a complete breakdown of the steps necessary to reproduce this and the versions required? I cannot reproduce the problem using RxJS `6.6.6` and TypeScript `4.2.3` - using `tsc` for the build. Also `Rx.d.ts` and the related files are only used if `rxjs-compat` is used. Is the problem reproducible only if that package is used?

**Comment by lildeadprince** [comment]
Reproduction is as little as I've mentioned originally: just two lines of code.
I've also created now a little repo with it here for you to check https://github.com/lildeadprince/rxjs-symbol-typedef
`npm run compile-default` for compilation with error
`npm run compile-workaround` for successful compilation using another import for missing declarations as workaround.


As for the `Rx.d.ts` I might've confused you, since it was just the first place I found the missing declaration and so I mentioned it. Type imports there very intricate and unfortunately I did not have enough time to investigate it originally. 

Correct narrowing down would be probably `'internal/symbol/observable'` for now.
